docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -49,17 +52,46 @@ module.exports = {
 
 In this example, there is a `Restaurant` model which contains the attributes `cover`, `name` and `description`.
 
+### Component's models
+
+It also exist another type of models named `components`. A component is a data structure that can be used in one or many other API's model. There is no lifecycle related, only a JSON file definition
+
+**Path —** `./components/default/simple.json`
+
+```json
+{
+  "connection": "default",
+  "collectionName": "components_default_simples",
+  "info": {
+    "name": "simple",
+    "icon": "arrow-circle-right"
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string"
+    }
+  }
+}
+```
+
+In this example, there is a `Simple` component which contains the attributes `name`. And the component is in the category `default`.
+
 ### Where are the models defined?
 
-The models are defined in each `./api/**/models/` folder. Every JavaScript or JSON file in these folders will be loaded as a model. They are also available through the `strapi.models` and `strapi.api.**.models` global variables. Usable everywhere in the project, they contain the ORM model object that they refer to. By convention, a model's name should be written in lowercase.
+For **Content Types**, models are defined in each `./api/**/models/` folder. Every JavaScript or JSON file in these folders will be loaded as a model. They are also available through the `strapi.models` and `strapi.api.**.models` global variables. Usable everywhere in the project, they contain the ORM model object that they refer to. By convention, a model's name should be written in lowercase.
```

Comment:
We shouldn't be documenting the use of `strapi.components` as the components models aren't made to be used individually.

Reply:
Can you precise please, I'm not sure to understand